Packs/CiscoUmbrellaReporting/Integrations/CiscoUmbrellaReporting/README.md

```diff
@@ -0,0 +1,1910 @@
+| limit | The maximum number of records to return from the collection. Limit default value is 50. If the page_size argument is set by the user then the limit argument will be ignored. | Optional | 
+| domains | A domain name or comma-separated list of domain names. | Optional | 
+| urls | A URL or comma-separated list of URLs. | Optional | 
+| ip | An IP address. | Optional | 
+| verdict | A verdict string. Possible values are: allowed, blocked, proxied. | Optional | 
+| sha256 | A SHA-256 hash. | Optional | 
+| amp_disposition | An Advanced Malware Protection (AMP) disposition string. Possible values are: clean, malicious, unknown. | Optional | 
+| identity_types | An identity type or comma-separated list of identity types. | Optional | 
+| page | The page number. Default is 1. | Optional | 
+| page_size | The number of requested results per page. Default is 50.| Optional | 
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| UmbrellaReporting.File.requests | Number | Number of requests. | 
+| UmbrellaReporting.File.identitycount | Number | Number of identities for entry. | 
+| UmbrellaReporting.File.sha256 | String | SHA256 for entry. | 
+| UmbrellaReporting.File.filenames | Unknown | Array of filenames for entry. | 
+| UmbrellaReporting.File.filetypes | Unknown | Array of file types for entry. | 
+| UmbrellaReporting.File.categories.id | Number | ID of the category. | 
+| UmbrellaReporting.File.categories.label | String | The human readable label of the category. | 
+| UmbrellaReporting.File.categories.type | String | The type of category. | 
+| UmbrellaReporting.File.categories.deprecated | Boolean | Whether the category is a legacy category. | 
+| UmbrellaReporting.File.categories.integration | Boolean | Whether the category is an integration. | 
+
+#### Command example
+```!umbrella-reporting-file-list limit=2```
+#### Context Example
+```json
+{
+    "UmbrellaReporting": {
+        "File": [
+            {
+                "categories": [
+                    {
+                        "deprecated": false,
+                        "id": 142,
+                        "integration": false,
+                        "label": "Online Meetings",
+                        "type": "content"
+                    },
+                    {
+                        "deprecated": false,
+                        "id": 148,
+                        "integration": false,
+                        "label": "Application",
+                        "type": "application"
+                    }
+                ],
+                "filenames": [
+                    "AnyDesk.exe"
+                ],
+                "filetypes": [],
+                "identitycount": 1,
+                "requests": 2,
+                "sha256": "94fe42af4a67ed5be45bd7913d8a8aebc4e35afddd5675d01bd37df8e9b399ae"
```

Comment:
```suggestion
                "sha256": "dummy_sha256"
```
Please apply to all sha256s in the readme file.

Reply:
@ShacharKidor Modified